subsys/net/lib/sockets/sockets.c

```diff
@@ -530,6 +539,8 @@ int zsock_accept_ctx(struct net_context *parent, struct sockaddr *addr,
 		if (net_pkt_eof(last_pkt)) {
 			sock_set_eof(ctx);
 			z_free_fd(fd);
+            zsock_flush_queue(ctx);
+            net_context_unref(ctx);
```

Comment:
There seems to be something wrong with indentation (or could be my browser too).